hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/SparkController.java

```diff
@@ -118,9 +118,8 @@ public Execution startJob(final Jobs job, final Users user)
 
     //If it is a notebook we need to convert it to a .py file every time the job is run
     if(appPath.endsWith(".ipynb")) {
-      int extensionIndex = appPath.lastIndexOf(".ipynb");
-      StringBuilder pathBuilder = new StringBuilder(appPath.substring(0, extensionIndex)).append("_job_" +
-              job.getName() + ".py");
+      String outPath = "hdfs:///Projects/" + job.getProject().getName() + "/" + Settings.PROJECT_STAGING_DIR;
```

Comment:
@robzor92 instead of `Projects` you can use `Settings.DIR_ROOT`

Reply:
Fixed